pnpm-workspace.yaml

```diff
@@ -1,2 +1,4 @@
 onlyBuiltDependencies:
   - esbuild
+packages:
+  - "./"
```

Comment:
please delete this changes from the file. This repository is not a monorepo. You should not add any settings for `packages`.